.changeset/early-ducks-stare.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-catalog-backend': minor
```

Comment:
Not a breaking change right? I don't think we need to treat the changed behaviour as a breaking change since it's just improving things :grin:

Reply:
The constructor for `DefaultProcessingDatabase` has a breaking change but perhaps it's considered a minor given it's mean to be used via the CatalogBuilder which is not breaking, am I right @Rugvip? :)

Reply:
```suggestion
'@backstage/plugin-catalog-backend': patch
```

Reply:
Discussed on discord, it's not exported :grin: